bundles/core/org.eclipse.smarthome.core.thing.xml.test/src/test/groovy/org/eclipse/smarthome/core/thing/xml/test/ThingTypesTest.groovy

```diff
@@ -71,6 +71,13 @@ class ThingTypesTest extends OSGiTest {
             assertThat size(), is(3)
             def colorChannel = it.find { it.id.equals("color") } as ChannelDefinition
             assertThat colorChannel, is(notNullValue())
+           
+            colorChannel.with {
+            	assertThat properties.size(), is(2)
+        		assertThat properties.get("key1"), is("value1")
+        		assertThat properties.get("key2"), is("value2")

```

Comment:
I know, we have no common groovy formatter, but here spaces and tabs are mixed. Could you change it to spaces, please.